.circleci/ci/deploy-ams.js

```diff
@@ -57,6 +57,12 @@ ci.dir('venia-store/cif-sample-project', () => {
     projectPom = projectPom.replace('<module>samplecontent</module>', '<module>samplecontent</module>\n        <module>dispatcher</module>')
     fs.writeFileSync('pom.xml', projectPom);
 
+    // Update dispatcher module pom.xml
+    let dispatcherPom = fs.readFileSync("dispatcher/pom.xml", "utf8");
+    let versionRegEx = new RegExp("<version>\\d\\.\\d\\.\\d</version>");
```

Comment:
I'm being picky, but you should change to `\\d+\\.\\d+\\.\\d+` to make sure we will support version digits > 9.